docs/ops.html

```diff
@@ -3956,25 +3956,31 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
     While the cluster is still in migration mode, it is possible to revert to ZK mode. In order to do this:
     <ol>
       <li>
-        For each KRaft broker:
+        If you have brokers in KRaft mode, for each one:
         <ul>
           <li>Stop the broker.</li>
           <li>Remove the __cluster_metadata directory on the broker.</li>
-          <li>Remove the <code>zookeeper.metadata.migration.enable</code> configuration and the KRaft controllers related configurations like <code>controller.quorum.voters</code>
-            and <code>controller.listener.names</code> from the broker configuration properties file.</li>
-          <li>Restart the broker in ZooKeeper mode.</li>
+          <li>Reconfigure the broker so that it will restart in ZK mode, by removing process.roles
+              and re-adding zookeeper.connect, and so forth.</li>
```

Comment:
Isn't this wrong? You can rollback when you are in "dual-write" and in this state you still have zookeeper.connect and you don't have process.roles yet.

Reply:
When the broker uses ZK but `zookeeper.metadata.migration.enable` = true, it is in "hybrid mode".

This is a subtype of ZK mode. I can clarify the language a bit to avoid confusion.